SEBrowser/Scripts/TSX/Components/EventSearch/EventSearchPreview/EventSearchCapBankAnalyticOverview.tsx

```diff
@@ -24,75 +24,56 @@
+
+interface ICapBankAnalytic {
+    ID: number,
+    Phase: string,
+    Status: string,
+    Operation: string,
+    Resonance: boolean,
+    CapBankHealth: string,
+    PreInsertionSwitch: string,
+    Restrike: string
 }
 
 const EventSearchCapBankAnalyticOverview: React.FC<SEBrowser.IWidget<any>> = (props) => {
-    const [tableRows, setTableRows] = React.useState<Array<JSX.Element>>([]);
+    const [data, setData] = React.useState<ICapBankAnalytic[]>([]);
     const service = new OpenSEEService();
 
     React.useEffect(() => {
         if (props.eventID >= 0)
-            createTableRows(props.eventID);
+            service.getCapBankAnalytic(props.eventID).then(setData);
     }, [props.eventID]);
 
-    function createTableRows(eventID: number) {
-        
-        service.getCapBankAnalytic(props.eventID).done(data => {
-            var rows = [];
 
-            for (var index = 0; index < data.length; ++index) {
-                var row = data[index];
-
-                rows.push(Row(row));
-            }
-
-            setTableRows(rows);
-        });
-    }
-
-        return (
-            <div className="card">
-                <div className="card-header">EPRI Capacitor Bank Analytic:</div>
-                <div className="card-body">
-                    <table className="table">
-                        <thead>
-                            <HeaderRow />
-                        </thead>
-                        <tbody>
-                            {tableRows}
-                        </tbody>
-                    </table>
-
-                </div>
+    return (
+        <div className="card" style={{ maxHeight: props.maxHeight ?? 500, overflowY: 'auto' }}>
+            <div className="card-header">EPRI Capacitor Bank Analytic:</div>
+            <div className="card-body">
+                <Table
+                    cols={[
+                        { key: 'Phase', field: 'Phase', label: 'Phase' },
+                        { key: 'Status', field: 'Status', label: 'Analysis Status' },
+                        { key: 'Operation', field: 'Operation', label: 'Capacitor Bank Operation' },
+                        { key: 'Resonance', field: 'Resonance', label: 'Resonance' },
+                        { key: 'Health', field: 'CapBankHealth', label: 'Capacitor Bank Health' },
+                        { key: 'Restrike', field: 'Restrike', label: 'Restrike' },
+                        { key: 'PIS', field: 'PreInsertionSwitch', label: 'PreInsertionSwitching Condition' }
+                    ]}
+                    data={data}
+                    onClick={() => { }}
+                    onSort={() => { }}
+                    sortKey={''}
+                    ascending={true}
+                    tableClass="table"
+                    theadStyle={{ fontSize: 'smaller', display: 'table', tableLayout: 'fixed', width: '100%', height: 50 }}
+                    tbodyStyle={{ display: 'table', overflowY: 'scroll', width: '100%' }}
```

Comment:
`display` should be `'block'`, not `'table'` in order for the card to scroll correctly.